js/index.js

```diff
@@ -36,3 +36,77 @@ for (let i = 0; i < skills.length; i++) {
   // Append the list item element to the unordered list
   skillsList.appendChild(skill);
 }
+
+// Select the message form by name
+const messageForm = document.forms.leave_message;
+
+// Add event listener for form submit
+messageForm.addEventListener('submit', function(event) {
+  // Prevent the form from submitting normally
+  event.preventDefault();
+
+  // Retrieve the form field values
+  const name = event.target.usersName.value;
+  const email = event.target.usersEmail.value;
+  let message = event.target.usersMessage.value;
+
+  // Log the form field values to the console
+  console.log('Name:', name);
+  console.log('Email:', email);
+  console.log('Message:', message);
+
+const messageSection = document.querySelector('#messages');
```

Comment:
messageSection and messageList are static.  They should be defined outside of the callback function.

Reply:
fixed